modular-docs-manual/content/topics/module_what-modular-documentation-is.adoc

```diff
@@ -3,6 +3,8 @@
 
 Modular documentation is documentation based on _modules_, which the writer combines into _assemblies_. An assembly can also include other assemblies.
 
+NOTE: A module cannot contain another module.
```

Comment:
My $0.02 is that this doesn't need to be a note. It can follow the perfectly clear previous sentence...maybe "However, a module cannot contain another module."

Reply:
+1, in fact would be better to do so for more emphasis, and would nix the note followed by important note.

Reply:
Fixed.